pkg/application/options.go

```diff
@@ -245,14 +233,69 @@ func WithLoggerSettingsFromConfig(app *App) {
 
 		loggerOptions := []mon.LoggerOption{
 			mon.WithLevel(settings.Level),
-			mon.WithFormat(settings.Format),
-			mon.WithTimestampFormat(settings.TimestampFormat),
+		}
+
+		loggerOptions, err := getLoggerHandlersFromConfigWithFallback(config, loggerOptions)
+		if err != nil {
+			return err
 		}
 
 		return logger.Option(loggerOptions...)
 	})
 }
 
+func getLoggerHandlersFromConfigWithFallback(config cfg.GosoConf, loggerOptions []mon.LoggerOption) ([]mon.LoggerOption, error) {
```

Comment:
i agree, that this is not optimal..
i would need some time to think about it.. does it have time until next year?

Reply:
Definitely, i do see that such big changes should be thought through, so we don't have to re-implement everything in a worst case scenario.